test/api/vclamp.cpp

```diff
@@ -32,11 +32,17 @@ int main(void) {
     nrn_init(3, argv.data());
 
     // load the stdrun library
-    char* temp_str = strdup("stdrun.hoc");
-    nrn_str_push(&temp_str);
-    nrn_function_call(nrn_symbol("load_file"), 1);
-    nrn_double_pop();
-    free(temp_str);
+    // Ensure If arguments are not correct we handle gracefully
+    auto res = nrn_function_call("load_file", NRN_NO_ARGS);
+    if (res.type == NrnResultType::NRN_ERR) {
+        std::cerr << "Err: " << nrn_stack_err() << std::endl;
+    }
+    // Do it right this time
+    nrn_function_call_s("load_file", "stdrun.hoc");
```

Comment:
`nrn_function_call_s` only works for one specific prototype. I don't think we want anything that specific in our API or in our examples.

Reply:
See comment above